test/parallel/test-tls-client-default-ciphers.js

```diff
@@ -40,7 +40,7 @@ function test1() {
   };
 
   try {
-    tls.connect(common.PORT);
+    tls.connect(0);
```

Comment:
Same: port 0 on `connect()` makes no sense. Comment needed if this change is correct.